nais-dev.yaml

```diff
@@ -81,3 +81,11 @@ spec:
       value: aapen-arbeidsrettetOppfolging-endringPaNyForVeileder-v1-q1
     - name: ENDRING_PA_MAL_TOPIC
       value: aapen-arbeidsrettetOppfolging-endringPaMal-v1-q1
+  azure:
+    application:
+      enabled: true
+  accessPolicy:
+    inbound:
+      rules:
+        - application: veilarbregistrering
+          namespace: paw
```

Comment:
Det er lagt inn config for dette lenger opp, mangler bare å legge inn inbound reglen for veilarbregistrering

Reply:
Aha, glemte å sjekke rebasen i farta